### DISCUSSION TITLE
Radio Beep stopped working

### DISCUSSION THREAD

**Discussion by ClydeHype** [IDI-Systems/acre2#1219] [Q&A] [answered]
Hello,

Some of my friends and I are using ACRE2 on our server and have noticed that some, if not all of us, have problems with the transmission beep that usually sounded when you pressed the "radio talk" button is gone.
Is there a fix for this, or is it a bug?

Best regards
Ricky

**Comment by PabstMirror** [comment, marked as answer]
Make sure your server.cfg has "b64" whitelisted in `allowedLoadFileExtensions[]` and `allowedPreprocessFileExtensions[]` if you are using that.
